src/core/Dropdown/Dropdown/Dropdown.tsx

```diff
@@ -241,37 +195,85 @@ export class Dropdown extends Component<DropdownProps> {
         id={id}
         {...passProps}
       >
-        <HtmlLabel
+        <LabelText
```

Comment:
I would consider adding a wrapper between the outermost span and label / other contents. So styles including display would be set on the new wrapper, but classnames, id and passprops left as is. This would allow setting margins, display: flex and so on for Dropdown without adding a wrapper or breaking styles.

Reply:
Good idea. Added a simple div as a wrapper.

Reply:
Actually, tested the previous version with `{{display: 'inline', margin: '50px', position: 'absolute', top: '100px', left: '300px'}}` and nothing seems to be broken. It might work with new version as well even without the wrapping div. There seems to be no reset styles for the original wrapper (If there were, those styles should now be applied to the new wrapping div to prevent an unintended override), so the additional div might be redundant.

Reply:
Should be ok as discussed.